DQMOffline/Configuration/python/DQMOffline_cff.py

```diff
@@ -153,7 +154,9 @@
 
 #Post sequences are automatically placed in the EndPath by ConfigBuilder if PAT is run.
 #miniAOD DQM sequences need to access the filter results.
-PostDQMOfflineMiniAOD = cms.Sequence(miniAODDQMSequence*jetMETDQMOfflineSourceMiniAOD*tracksDQMMiniAOD*topPhysicsminiAOD)
+from DQMOffline.Muon.miniAOD_cff import * 
+
+PostDQMOfflineMiniAOD = cms.Sequence(miniAODDQMSequence*jetMETDQMOfflineSourceMiniAOD*tracksDQMMiniAOD*topPhysicsminiAOD*muonMonitors_miniAOD*MuonMiniAOD)
```

Comment:
Maybe you can already confirm is the same is true for the muonMonitors_miniAOD sequence? (otherwise I'll look at that next)